xml/System.Threading.Channels/ChannelWriter`1.xml

```diff
@@ -159,8 +159,7 @@
       <Docs>
         <param name="item">The item to write.</param>
         <summary>Attempts to write the specified item to the channel.</summary>
-        <returns>
-          <see langword="true" /> if the item was written; otherwise, <see langword="false" />.</returns>
+        <returns><see langword="true" /> if the item was written; otherwise, <see langword="false" />.</returns>
```

Comment:
The next time the CI runs, this will go back to being on a newline, so it's best to undo this change.

Reply:
It just seemed odd that the opening tag was on its own line, but the the closing `</returns>` tag wasn't. It also looked inconsistent with the other sections in the file.

Would it be better to add a newline before the closing tag, or just leave it exactly as it was?

Reply:
The CI that we run to ingest the APIs puts anything that starts with `<` on a newline, so it's best to leave it just the way it was.